### DISCUSSION TITLE
Template support no longer available since version 3

### DISCUSSION THREAD

**Discussion by rventurelli** [dineug/erd-editor#366] [General]
Is template support no longer available since version 3?

It was a killing feature... 😢

**Comment by dineug** [comment]
For most cases, the basic functionality will be sufficient.
What kind of template do you need?
<img width="519" alt="스크린샷 2024-05-16 232302" src="https://github.com/dineug/erd-editor/assets/45829489/e2265b4c-b90a-4d05-ae35-a9a6c2b7ae5e">

**Comment by rventurelli** [comment]
In the previous version, we could create a custom code generator to produce a JSON file that was then used by our code generators:

![image](https://github.com/dineug/erd-editor/assets/1785571/b822986d-ffad-4e16-9f77-57771750d944)

**Reply by dineug** [reply]
Currently, there are no plans for support. Since the *.erd.json structure is simple, if needed, we recommend creating your own CLI script.
